.github/workflows/build-package.yaml

```diff
@@ -0,0 +1,237 @@
+# This automation builds 3p packages based on a PR
+
+name: Build 3P Packages
+
+on:
+  pull_request:

```

Comment:
Should we add `types: [opened, reopened]` to limit when we trigger this workflow

Also do we want to enable running this on PR from a fork or just supported branches?

Reply:
We should rebuild on changes, not just when the PR is opened, but good callout to limit builds. I'll see if we can put the build portion behind the github dialog for environments. I'll also see about using the `workflow_dispatch` trigger so that someone in a fork can trigger manually.

Reply:
Actually, this won't retrigger unless `package_build_list_host_*.json` is changed. If someone is making a further change to their PR that doesn't involve editing `package_build_list_host_*.json` again, but requires rebuild, they can just do a re-run of the build job, since it'll pull from latest